### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by Motorrat** [rasbt/musicmood#2]
Sebastian, isn't your model overfitting as the recall falls sharply when you going from train to test data.

**Comment by rasbt** [comment]
Yup, I agree! I got more stable results with regularized logistic regression but couldn't find a way to get a lower train/test discrepancy with any of the naive Bayes models. I started to collect more data but never got around re-training the models, something that's still on my to-do list for some point in future!
